packages/tools/kadena-cli/src/account/commands/accountFund.ts

```diff
@@ -34,6 +36,11 @@ export const createAccountFundCommand = createCommand(
     // deployDevnet(),
   ],
   async (option) => {
+    const allAccounts = await getAllAccounts();
+    if (allAccounts.length === 0) {
```

Comment:
seems a bit odd to get all accounts just to check if there are any. Wasn't there some `ensureAccountsExists` method or something that was used inside `getAllAccounts()` before?

Reply:
yes, makes sense! updated